app/assets/stylesheets/blacklight_gallery/_masonry.scss

```diff
@@ -0,0 +1,44 @@
+.view-type-masonry {
+  background-image: asset-url('icon-masonry.png') !important;
+  background-position: center;
+  background-repeat: no-repeat;
+  .view-icon-masonry {
+    padding-right: 12px;
```

Comment:
This is a one-off padding necessary for the icon to look right (since we're faking a font-icon w/ a real .png).  I believe this would have an unintended affect on this button (i.e. making it much wider than the other buttons).

Perhaps this says that we shouldn't be faking this icon, but I'm just working with what I have.